trappy/ftrace.py

```diff
@@ -85,6 +133,10 @@ def __init__(self, name="", normalize_time=True, scope="all",
         if normalize_time:
             self.normalize_time()
 
+        _create_trace_cache(self.trace_path)
+        for trace_class in self.trace_classes:
+            trace_class.write_csv(cache_path + '/' + trace_class.unique_word[:-1] + '.csv')
```

Comment:
- `os.path.join`
- The creation of the CSV path is duplicated here, should probably put it in a method